test/e2e/runner/exec.go

```diff
@@ -39,20 +39,20 @@ func execVerbose(args ...string) error {
 // execCompose runs a Docker Compose command for a testnet.
 func execCompose(dir string, args ...string) error {
 	return exec(append(
-		[]string{"docker-compose", "-f", filepath.Join(dir, "docker-compose.yml")},
+		[]string{"docker", "compose", "-f", filepath.Join(dir, "docker-compose.yml")},
```

Comment:
for some reason my brain is telling me that we might need to use "docker compose" instead of "docker" "compose"

I forget tho